databases/cockroachdb/tests/prisma.test.ts

```
@@ -1,6 +1,13 @@
-import { PrismaClient } from '@prisma/client'
+import { PrismaClient } from '../prisma/generated/client'
+import { PrismaPg } from '@prisma/adapter-pg'
+import 'dotenv/config'
 
-export const prisma = new PrismaClient()
+const adapter = new PrismaPg({
+  connectionString: process.env.DATABASE_URL,
+})
+export const prisma = new PrismaClient({
+  adapter,
+})
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)